client/stylesheets/_toCat.scss

```diff
@@ -3678,6 +3678,22 @@ On click the images will be displayed at normal size to complete the effect
     border: 2px solid #f8504b;
 }
 
+.profileHistoryCardUl {
+    background-color: gainsboro;
+}
+
+.profileHistoryTimestamp {
+    border: none !important;
+}
+
+.profileHistoryTimestamp::hover {
+      border: none !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 6 spaces